src/dsimhost_fpga.py

```diff
@@ -195,13 +195,9 @@ def setup_tengbes(self):
         gbes_per_pol = 2        # Hardcoded assumption
         num_pols = num_tengbes // gbes_per_pol
 
-        ip_bits = start_ip.split('.')
-        ipbase = int(ip_bits[3])
-        mac_bits = start_mac.split(':')
-        macbase = int(mac_bits[5])
         for ctr in range(num_tengbes):
-            mac = '%s:%d' % (':'.join(mac_bits[0:5]), macbase + ctr)
-            ip = '%s.%s.%s.%d' % (ip_bits[0], ip_bits[1], ip_bits[2], ipbase + ctr)
+            mac = str(tengbe.Mac(start_mac + ctr))
+            ip = str(tengbe.IpAddress(start_ip + ctr))
```

Comment:
So here I set mac and ip to strings for gbe.setup(). It is a small change in the rest of the files but I was thinking maybe it's better to change gbe.setup() so it accepts ints?